### DISCUSSION TITLE
Is this still current and working?

### DISCUSSION THREAD

**Discussion by Raging-Beard** [jamescl604/MSFSVoiceAttackPlugin#15] [General]
I see this was released over a year ago and has a couple of unanswered discussions. I was wondering if it still works after all the sim updates, any better alternatives? Thanks.

**Comment by KevinMicknick** [comment]
I have not been able to get it to work. I get the vars sent to it and get back everytime the var is set to zero

**Comment by CardosiDSilva** [comment]
Hi folks

Got the plugin installed but VA is showing the below error:

"11:12:41.762 NetworkCommsDotNet.ConnectionSetupException: Error during TCP connection establish with destination ([TCP-E-S] 0.0.0.0:0 -> 127.0.0.1:8181). 
Destination may not be listening or connect timed out. System.Net.Sockets.SocketException (0x80004005): No connection could be made because the target machine actively refused it 127.0.0.1:8181    
at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)    at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)    at System.Net.Sockets.Tcp..."

I found a CFG file within the msfs-agent folder and showing address Address=127.0.0.1 (default), assuming that it would work changing to my own local address (192.XXX.X.XX). Clearly hasn't worked, haha. Pluging still not visible, my other 2 plugins for VA are just fine and showing up on the list.

Halp please :)

Cheers

**Reply by CardosiDSilva** [reply]
Solved :) apologies for the trouble.
